azure-pipelines.yml

```diff
@@ -10,9 +10,6 @@ pool:
   vmImage: 'vs2017-win2016'
 strategy:
   matrix:
-    Python35:
-      python.version: '3.5'
-      TOXENV: py35
```

Comment:
Python 3.5 is deprecated but still supported by Qiskit until the upstream python EoL (which is in Sept. iirc) we shouldn't drop this testing (especially just to use f strings which can easily be changed).

Reply:
True.  We have internally standardized on using f-strings whenever possible for the clarity, but support of 3.5 seems a fine enough reason to do a bit of string concatenation.